Zend/Optimizer/block_pass.c

```diff
@@ -257,46 +257,54 @@ static void zend_optimize_block(zend_basic_block *block, zend_op_array *op_array
 				break;
 
 			case ZEND_FREE:
+				src = VAR_SOURCE(opline->op1);
+				if (!src) {
```

Comment:
Would it be possible to keep the redundant `src` check in the two  ìf`s to reduce the diff? IMO, refactoring that causes big whitespace changes should be done separately, if at all.

Reply:
Sure, I guess the compiler can probably get rid of the redundant check (hopefully).